### PR TITLE
fixed text layout

### PR DESCRIPTION
**Pull request by syedowaisali90**
Hi @vincent-karuri I have created a PR for review.

### CODE REVIEW

File: opensrp-app/res/layout/smart_register_pagination.xml [hunk 1]
```diff
@@ -1,39 +1,43 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (vincent-karuri) [inline opensrp-app/res/layout/smart_register_pagination.xml:3]: Not sure why we have this large diff here?

File: opensrp-app/res/values/dimens.xml [hunk 2]
```diff
@@ -1,132 +1,134 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
```
Comment (vincent-karuri) [inline opensrp-app/res/values/dimens.xml:2]: Not sure why we have this large diff here?

File: opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java [hunk 3]
```diff
@@ -399,9 +403,14 @@ private int getTotalcount() {
...
 
     public void refresh() {
+        int textSize = getResources().getDimensionPixelSize(R.dimen.pagination_page_info_size);
+        int startIndex = getResources().getString(R.string.str_page_info).indexOf("{");
         pageInfoView.setText(
                 format(getResources().getString(R.string.str_page_info), (getCurrentPageCount()),
                         getTotalcount()));
+        SpannableString span = new SpannableString(pageInfoView.getText());
+        span.setSpan(new AbsoluteSizeSpan(textSize), startIndex, pageInfoView.getText().length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+        pageInfoView.setText(span);
```
Comment (vincent-karuri) [inline opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java:413]: Any reason for this extra logic?

File: opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java [hunk 4]
```diff
@@ -1,647 +1,654 @@
-package org.smartregister.view.activity;
```
Comment (vincent-karuri) [inline opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java:1]: Not sure why we have this large diff here?

File: opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java [hunk 5]
```diff
@@ -439,8 +443,14 @@ private int getCurrentPageCount() {
         }
 
         public void refresh() {
+            int textSize = getResources().getDimensionPixelSize(R.dimen.pagination_page_info_size);
+            int startIndex = getResources().getString(R.string.str_page_info).indexOf("{");
```
Comment (vincent-karuri) [inline opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java:447]: Any reason for this additional logic?

File: opensrp-app/res/layout/smart_register_pagination.xml (hunk 1 above)
Comment (syedowaisali90) [reply opensrp-app/res/layout/smart_register_pagination.xml:3]: Actually, it looks like a large change but it's the minimal change you can check the diff

File: opensrp-app/res/values/dimens.xml (hunk 2 above)
Comment (syedowaisali90) [reply opensrp-app/res/values/dimens.xml:2]: same as above

File: opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java (hunk 3 above)
Comment (syedowaisali90) [reply opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java:413]: just for reduce the page info like `1 of 3`

File: opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java (hunk 4 above)
Comment (syedowaisali90) [reply opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java:1]: https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551815077

File: opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java (hunk 5 above)
Comment (syedowaisali90) [reply opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java:447]: https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551821442

File: opensrp-app/res/layout/smart_register_pagination.xml (hunk 1 above)
Comment (vincent-karuri) [reply opensrp-app/res/layout/smart_register_pagination.xml:3]: If it's a formatting issue, please revert to the previous format.

File: opensrp-app/res/values/dimens.xml (hunk 2 above)
Comment (vincent-karuri) [reply opensrp-app/res/values/dimens.xml:2]: https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551863135

File: opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java (hunk 4 above)
Comment (vincent-karuri) [reply opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java:1]: https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551863135

File: opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java (hunk 3 above)
Comment (vincent-karuri) [reply opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java:413]: I think the requirement was to allow the developer to override a method that determines how the pagination information is displayed? 

The size shouldn't be reduced in core.

File: opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java (hunk 5 above)
Comment (vincent-karuri) [reply opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java:447]: https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551864293

File: opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java (hunk 3 above)
Comment (syedowaisali90) [reply opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java:413]: fixed.

File: opensrp-app/res/layout/smart_register_pagination.xml (hunk 1 above)
Comment (syedowaisali90) [reply opensrp-app/res/layout/smart_register_pagination.xml:3]: locally not showing any formatting issue

**Review by vincent-karuri — DISMISSED** [review]

**Review by vincent-karuri — APPROVED** [review]

**Review by vincent-karuri — APPROVED** [review]

**Pull request merged**